src/home_robot_sim/home_robot_sim/env/habitat_objectnav_env/visualizer.py

```diff
@@ -70,80 +70,7 @@ def __init__(self, config, dataset=None):
             or "hm3d" in self.episodes_data_path
             or "mp3d" in self.episodes_data_path
         )
-        if "hm3d" in self.episodes_data_path:
```

Comment:
All of this is just unnecessary now?

Reply:
It comes from the envs. For objectnav, this was already present in HabitatObjectNavEnv. I tested on hm3d and it doesn't break

Reply:
Great!